articles/migrations/0002_auto_20240220_1730.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+from django.core.management import call_command
+
+
+def load_initial_data(apps, schema_editor):
+    call_command(
+        "loaddata",
+        "article_status.json",
+        "request_status.json",
```

Comment:
Writing more than one data migrations within one file is not a good practice. Suppose you're in a situation where you need to rollback the data migration which should only effect the ReviewRequestStatus table and you need to keep all the data in ArticleStatus table as it is. With your approach, you can't do that. If you rollback(unapply) this data migration, all the data within ArticleStatus & ReviewRequestStatus will be vanished. But if you write two data migrations to populate data to these two tables separately, you can unapply any data migration at any time without affecting the other. Please ask me if you don't understand this.